.storybook/stories/components/Badges/IconBadge.stories.js

```diff
@@ -0,0 +1,33 @@
+import { IconBadge } from 'components/shared/IconBadge'
+import title from '../../../utils/titles'
```

Comment:
See comment below about moving this to the `.storybook` folder. You can make a `utils` folder there if you would like.

Reply:
Could you please clarify this?

Reply:
It's already in the `.storybook/utils/` in the folder. I'll create alias. Thank you!